### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by Donkey-Doug** [deckerst/aves#138]
Add a changelog to the releases. It is fine if the changes are only on github and not in the app. Fdroid/play store should also be able to show changes.

Update: nevermind, there is a changelog in the play store & on github. Perhaps consider adding it to izzydroid as well.

**Comment by D3ZOXY** [comment]
@deckerst 
Can you please write the changelog here?
![Changelog](https://user-images.githubusercontent.com/47178587/144319963-b354c263-72cb-4b33-bb2a-c5949cb41e47.png)

**Comment by Donkey-Doug** [comment]
I could not find the changelog on github too. I just read on the play store where it said "see github for full changelog"

**Comment by D3ZOXY** [comment]
@Donkey-Doug 
https://github.com/deckerst/aves/commit/a2b84a66090de60cec8510f8ebb27278fbab7bbd

**Comment by deckerst** [comment]
The "changelog on GitHub" I'm referring to is [this one](https://github.com/deckerst/aves/blob/develop/CHANGELOG.md). It's a standard location, but I'll put a link to it in the readme, and maybe in the app too.

@Donkey-Doug
For F-Droid, I followed some instructions from Izzy to provide a changelog per release. But it didn't work as expected for the previous release (v1.5.6), so I dropped it for this release (v1.5.7). Anyway, I'd rather have the reference changelog in one place, the one I linked above.

@D3ZOXY
I'll see what I can do to link changes from the release page.
